core/server/worker/src/main/java/alluxio/worker/block/BlockMasterClient.java

```diff
@@ -41,7 +46,8 @@
  */
 @ThreadSafe
 public final class BlockMasterClient extends AbstractMasterClient {
-  private BlockMasterWorkerService.Client mClient = null;
+  // private BlockMasterWorkerService.Client mClient = null;
```

Comment:
remove

Reply:
Being addressed as part of boot-up and security work.